lib/wibox/widget/graph.lua

```diff
@@ -343,7 +343,7 @@ prop_fallbacks.nan_color = build_fallback_nan_color()
 --
 
 local function graph_gather_drawn_values_num_stats(self, new_value)
-    if not (new_value >= 0) then
+    if not (new_value >= 0) then --luacheck: ignore 581
```

Comment:
I'd add a comment that this specific expression is required to include the check for NaN.

Reply:
There's already a comment 1 line below

Reply:
That comment mentions that NaN is intended to be caught, but not how the linting rule has to be ignored because its suggested alternative doesn't actually do the same thing.

Reply:
Added. After researching, apparently `luacheck` did that on purpose and adding `--luacheck: ignore 581` is the official solution. They don't consider this a bug.